calendar.js

```diff
@@ -797,6 +806,9 @@ var cal = {
 			cal.eventsView.classList.add("ninja");
 			cal.getExport.classList.remove("ninja");
 			document.querySelector("#exportData").textContent = icsString;
+			if (event.length == 1) {

```

Comment:
I'd prefer to use === here

Reply:
you know that you can do suggestions in GitHub?
```suggestion
			if (event.length === 1) {

```